src/main/java/org/jabref/gui/importer/ImportFormats.java

```diff
@@ -67,12 +67,12 @@ public void actionPerformed(ActionEvent e) {
                 List<FileType> extensions = importers.stream().map(Importer::getFileType)
                         .collect(Collectors.toList());
                 FileChooser.ExtensionFilter allImports = FileFilterConverter.forAllImporters(importers);
-
                 FileDialogConfiguration fileDialogConfiguration = new FileDialogConfiguration.Builder()
                         .addExtensionFilter(allImports)
                         .addExtensionFilters(extensions)
                         .withInitialDirectory(Globals.prefs.get(JabRefPreferences.IMPORT_WORKING_DIRECTORY))
                         .build();
+                fileDialogConfiguration.getExtensionFilters().add(new FileChooser.ExtensionFilter("Any file", "*.*"));
```

Comment:
If you could just wrap that "Any file" in Localization.lang, I would be happy.
Otherwise LGTM! :+1:

Reply:
@Siedlerchr With this addition, is
https://github.com/JabRef/jabref/blob/58c2c7aa941744d9341454bfb5eeabeed77ab54e/src/main/java/org/jabref/gui/util/FileFilterConverter.java#L47-L49
now obsolete?

Reply:
Still a little difference, but you are in general right.

Reply:
So what exactly is the difference? Should we just remove the old one?

Reply:
All available import formats  = All supported file extension
Any file = Any extension not registered as default